src/ansi-c/scanner.l

```diff
@@ -217,10 +217,13 @@ hexfloat1       "0"[xX]{hexdigit}*"."{hexdigit}+[pP][+-]?{integer}
 hexfloat2       "0"[xX]{hexdigit}*"."[pP][+-]?{integer}
 hexfloat3       "0"[xX]{hexdigit}*[pP][+-]?{integer}
 float_suffix    [fFlLiIjJ]*
-gcc_ext_float_suffix    [wWqQ]|[dD][fFdDlL]?|"f16"|"f32"|"f64"|"f128"|"f32x"|"f64x"|"f128x"
+clang_ext_float_suffix [qQ]|"f16"|"F16"
```

Comment:
I believe these are standardised in the C++ standards; e.g., `bf16` is C++23. They are not clang extensions.

Reply:
GCC and Clang support these at varying versions, and while the standard appears to specify them I wasn't able to make them work when using `-std=c++23` (Clang) or `/std:c++latest` (MSVC).